### DISCUSSION TITLE
Patch Firmware

### DISCUSSION THREAD

**Discussion by Firdaus-gh** [egzumer/uv-k5-firmware-custom#336] [General]
I'm trying to patch egzumer v.021 with Custom Bootscreen but facing warning like this, pls adv the solution to solve this, thanks

" CRC check passed...
Detected firmware version: *EGZUMER v0.21
Firmware uses 100.65% of available memory (61841/61439 bytes).
WARNING: Firmware is too large and WILL NOT WORK!
Try disabling mods that take up extra memory. "

![image](https://github.com/egzumer/uv-k5-firmware-custom/assets/152011219/637f7911-22ab-43ae-8ff2-e5a0f2400726)

**Comment by OE1MWW** [comment]
you having the answer to your question on your own screen. The Boot Screen you created is using more memory space than available. 100.65% is a tick more than 100%. So, either modify it to fit into 100% (or better 99.99%), or you have to disable some functions that using firmware space.

**Comment by egzumer** [comment]
Go and educate those that recomended you the uvmod, you cannot use that tool for anything other than quansheng original v26 firmware.
